addons/binding/org.openhab.binding.xiaomivacuum/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="lib" path="lib/bcpkix-jdk15on-1.50.jar"/>
```

Comment:
This is because there is suppose to be PKCS7Padding in the en-/decrypting, which is not supported by oracle. bouncycastle is supporting it. 
I now applied PKCS5Padding, which seems to work as well. Need to deeper understand what is the precise difference between PKCS5Padding & PKCS7Padding to determine if I can drop it.